Bottom: dd19ebe04611b504b61fd0996bcd9f189b2fc290
Top:    0f36c6b08655477e229a50c8171ebcb9e86596d5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 21:05:56 +0100

Remove hack in draw.c which broke PS (and probably other) output

The set_layer call hacked into DrawLayerGroup to reset compositing tricked
the PS exporter to re-open and overwrite output in the main layer output
file during the fab drawing output.


---

diff --git a/src/draw.c b/src/draw.c
index 00d2785..c8599fd 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -701,9 +701,6 @@ DrawLayerGroup (int group, const BoxType *drawn_area)
         rv = 0;
       if (layernum < max_copper_layer && Layer->On)
         dapi->draw_layer (Layer, drawn_area, NULL);
-
-      if (gui->gui) /* HACK: Subcomposite each layer in a layer group separately */
-        gui->set_layer (0, group, 0);
     }
   if (n_entries > 1)
     rv = 1;
